docker-compose.yml

```diff
@@ -27,6 +27,7 @@ services:
     environment:
       MYSQL_DATABASE: canopeum_db
       MYSQL_HOST_CANOPEUM: /run/secrets/MYSQL_HOST_CANOPEUM
+      MYSQL_PORT_CANOPEUM: /run/secrets/MYSQL_PORT_CANOPEUM
```

Comment:
Setup changes are preferable as a different PR. If simple enough I can almost immediately approve those. (and I agree with this change, more control for the local dev and it doesn't hurt anything)

Reply:
See this [PR](https://github.com/BesLogic/releaf-canopeum/pull/204)